imdown-mobile/backend/backend.tsx

```diff
@@ -0,0 +1,24 @@
+import AsyncStorage from '@react-native-community/async-storage';
+
+export const BACKEND_URL = "http://localhost:5000/"
+
+export const callBackend = async (endpoint: string, init?: RequestInit) => {
+    if (init === undefined) {
+        init = {}
+    }
```

Comment:
I think you could probably just have
```
export const callBackend = async (endpoint: string, init: RequestInit = {}) => {
```
Also indentation is four spaces for this file but two spaces for our other one.

Reply:
I didn't even know you could do that, nice. And good catch my editor was being weird - everything should be 4 spaces now